homeassistant/components/simplisafe/__init__.py

```diff
@@ -201,10 +201,19 @@ def _async_save_refresh_token(hass, config_entry, token):
 
 async def async_setup_entry(hass, config_entry):
     """Set up SimpliSafe as config entry."""
+    entry_updates = {}
     if not config_entry.unique_id:
-        hass.config_entries.async_update_entry(
-            config_entry, unique_id=config_entry.data[CONF_USERNAME]
-        )
+        # If the config entry doesn't already have a unique ID, set one:
+        entry_updates["unique_id"] = config_entry.data[CONF_USERNAME]
+    if not config_entry.options:
```

Comment:
I'd appreciate feedback on this, in particular. It seems like an unnecessary step to save a code in the config entry data, only to later pop it out and move it to `options`, but I didn't see any examples showing something more streamlined.

Reply:
We want to avoid storing defaults in options, so that defaults can change overtime.

Reply:
How does 4875c0749cfb look?

Reply:
How does e38726fe10cc look?